src/command/defaults/TimeCommand.php

```diff
@@ -98,48 +95,30 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 				return true;
 			}
 
-			switch($args[1]){
-				case "day":
-					$value = World::TIME_DAY;
-					break;
-				case "noon":
-					$value = World::TIME_NOON;
-					break;
-				case "sunset":
-					$value = World::TIME_SUNSET;
-					break;
-				case "night":
-					$value = World::TIME_NIGHT;
-					break;
-				case "midnight":
-					$value = World::TIME_MIDNIGHT;
-					break;
-				case "sunrise":
-					$value = World::TIME_SUNRISE;
-					break;
-				default:
-					$value = $this->getInteger($sender, $args[1], 0);
-					break;
-			}
+			$value = match($args[1]){
+				"day" => World::TIME_DAY,
+				"noon" => World::TIME_NOON,
+				"sunset" => World::TIME_SUNSET,
+				"night" => World::TIME_NIGHT,
+				"midnight" => World::TIME_MIDNIGHT,
+				"sunrise" => World::TIME_SUNRISE,
+				default => $this->getInteger($sender, $args[1], 0),
+			};
```

Comment:
This change is unrelated.

Reply:
I didn't think it was necessary to keep the old code.

Reply:
It's also not necessary to change it, and doesn't contribute to the goal of the PR. Don't change unrelated things.

Reply:
As I said, it's not adding to the goal of the PR. It's just polluting your diff.